// _@vuepress_core@1.0.3@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "C:\\Users\\刘\\Desktop\\llblogs\\node_modules\\_@vuepress_core@1.0.3@@vuepress\\core\\lib\\client\\components\\GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-7b842e87",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7b842e87").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-33898df2",
    path: "/guide/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-33898df2").then(next)
    },
  },
  {
    path: "/guide/index.html",
    redirect: "/guide/"
  },
  {
    path: "/guide/guide.html",
    redirect: "/guide/"
  },
  {
    name: "v-5d463dbe",
    path: "/2019-06-21-deep-in-koa/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5d463dbe").then(next)
    },
  },
  {
    path: "/2019-06-21-deep-in-koa/index.html",
    redirect: "/2019-06-21-deep-in-koa/"
  },
  {
    path: "/js/01js.html",
    redirect: "/2019-06-21-deep-in-koa/"
  },
  {
    name: "v-06e86fc1",
    path: "/js/02js.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-06e86fc1").then(next)
    },
  },
  {
    name: "v-626e1470",
    path: "/qqq/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-626e1470").then(next)
    },
  },
  {
    path: "/qqq/index.html",
    redirect: "/qqq/"
  },
  {
    name: "v-104b0c8a",
    path: "/js/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-104b0c8a").then(next)
    },
  },
  {
    path: "/js/index.html",
    redirect: "/js/"
  },
  {
    name: "v-265b42b2",
    path: "/qqq/123.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-265b42b2").then(next)
    },
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
